### PR TITLE
Arcyd: add file handler to compress rotated debug logs

### LINKED ISSUE #18: Arcyd: var/log/debug rotates pretty fast, please consider zipping
For a large arcyd installation, the `var/log/debug` log file can rotate completely multiple times per day. This means that it's not possible to look into events from previous days.

Please consider zipping the rotated logs so that we can afford to store more logs.

### PR DESCRIPTION
**Pull request by cs-shadow**
Fixes #18

### CODE REVIEW

**Comment by aevri** [comment]
This is looking good! I especially like the test driver :)

Please could you also include some detail in the commit message about why you are making the change (could basically copy text from the original issue).

**Comment by cs-shadow** [comment]
Used the helper function to write files. Also added a more descriptive commit message. At the time of creating this PR, I was not fully aware of the standards and hence forgot to add the message, should be fine now.

I also added a note that this commit in itself will not change the amount of logs available or stored but will only reduce the size. Increasing the number of rotated logs stored should be a follow up to this.

File: py/phl/phlsys_compressedlogging__t.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    def test_B_rotation(self):
+        with phlsys_fs.chtmpdir_context():
+            debug_handler = _MAKE_HANDLER(
+                'testfile')
+            content = 'Hello World, this is a test for the rotator.'
+
+            phlsys_fs.write_text_file('testfile', content)
+            with phlsys_fs.write_file_lock_context('testfile') as src:
+                src.write(content)
```
Comment (aevri) [inline py/phl/phlsys_compressedlogging__t.py]: I think you accidentally didn't delete lines 76, 77 :)

Comment (cs-shadow) [inline py/phl/phlsys_compressedlogging__t.py]: Sorry, that was my bad. Should be fine now. :)

**Comment by aevri** [comment]
Aha, quick turnaround :) Will look again, thanks!

**Comment by aevri** [comment]
Landed at d4c96c5, thanks!

**Pull request closed without merging**